client/blocks/follow-button/button.jsx

```diff
@@ -33,12 +33,8 @@ class FollowButton extends React.Component {
 		disabled: false,
 	};
 
-	UNSAFE_componentWillMount() {
-		this.strings = {
-			FOLLOW: this.props.translate( 'Follow' ),
-			FOLLOWING: this.props.translate( 'Following' ),
-		};
-	}
+	FOLLOW = this.props.translate( 'Follow' );
```

Comment:
Theoretically, won't this be evaluated only the first time, and if the locale changes in runtime, they won't be updated? Also, we don't benefit anything by declaring those as instance properties.

To get around it, I'd suggest calling `this.props.translate()` as we usually do in the render method, rather than deferring those to object instance properties.

Reply:
> Theoretically, won't this be evaluated only the first time

I think that should be the same as before, no?

> I'd suggest calling this.props.translate() as we usually do in the render method

Agreed, calling `translate` directly makes better sense. No need to declare those as instance properties.

Reply:
I thought about this too but isn't it because it's an instance property that it will be re-evaluated each time the class is instantiated?

```
> class Test {
... prop = (() => console.log('hello'))();
... }
undefined
> new Test
hello
Test { prop: undefined }
> new Test
hello
Test { prop: undefined }
> new Test
hello
Test { prop: undefined }
```

Reply:
Yes, but isn't the new component instance created only when mounting the component? And because changing the locale doesn't guarantee that a new instance will be created, then this won't be re-evaluated on re-render?

Reply:
Any yeah, in any case as @flootr said, no need to indirect those calls.

Reply:
Ah yes, that would be a problem.

In any case moving it to render solves all of these 🙂